CalibFormats/SiStripObjects/test/testSiStripHashedDetId.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ function die { echo $1: status $2; exit $2; }
+
+if [ "${SCRAM_TEST_NAME}" != "" ] ; then
+  mkdir ${SCRAM_TEST_NAME}
+  cd ${SCRAM_TEST_NAME}
+fi
```

Comment:
@smuzaffar Do I remember correctly that nowadays SCRAM does effectively the same automatically?